intro_to_r/intro_to_r.Rmd

```diff
@@ -174,64 +183,67 @@ Now, suppose we want to plot the total number of baptisms.  To compute this, we
 could use the fact that R is really just a big calculator. We can type in 
 mathematical expressions like
 
-```{r calc-total-bapt-numbers, eval=FALSE}
+```{r calc-total-bapt-numbers}
 5218 + 4683
 ```
 
 to see the total number of baptisms in 1629. We could repeat this once for each 
-year, but there is a faster way. If we add the vector for baptisms for boys to 
+year, but there is a faster way. If we add the variable for baptisms for boys to 
 that of girls, R will compute all sums simultaneously.
 
-```{r calc-total-bapt-vars, eval=FALSE}
-arbuthnot$boys + arbuthnot$girls
+```{r calc-total-bapt-vars}
+arbuthnot %>%
+  mutate(total = boys + girls)
 ```
 
-What you will see are 82 numbers (in that packed display, because we aren’t 
-looking at a data frame here), each one representing the sum we’re after. Take a
-look at a few of them and verify that they are right.
+Again, we can read this chunk of code by saying, 'take the `arbuthnot` data, and then mutate it so there is an additional column called total, which is comprised of the columns `boys` and `girls` added together.'
+
+You'll see that there is now a new column called "total" that has been tacked on
+to the data frame. What "total" contains is a list of 82 numbers, each one representing the sum we’re after. Take a look at a few of them and verify that they are right.
 
-We'll be using this new vector to generate some plots, so we'll want to save it 
-as a permanent column in our data frame.
+While this output looks like our original dataset with an additional column added, it was just temporarily shown to us in the console. In order to use that column again, we have to save it. 
 
-```{r calc-total-bapt-vars-save, eval=FALSE}
-arbuthnot <- mutate(arbuthnot, total = boys + girls)
+```{r calc-total-bapt-vars-save}
+arbuthnot <- 
+  arbuthnot %>%
+  mutate(total = boys + girls)
 ```
 
```

Comment:
There's some text about this already in the lab, both before that chunk (line 204) and below (line 212). In my opinion, updating `arbuthnot` with more variables makes sense, as I don't want a huge number of slightly different data frames hanging around in my environment. However, I just ran through a (modified) version of this lab yesterday, and students often forgot to put a name in for their new variable and ended up with a variable named, for example `boys/(boys+girls)` which is hard to work with!